### PR TITLE
Add event tracking in GA #92

### LINKED ISSUE #92: Add event tracking in GA
We want to adopt [Event Tracking](https://developers.google.com/analytics/devguides/collection/analyticsjs/events) for certain actions, such as:

 * Clicked "Sign Up"
 * Clicked "Log In"

For each of these, we want to track as much metadata as possible, including:

 * What page the user was on
 * Which element (in case there are multiple of the same on the same page)
 * If it's something like the pricing page, we also need to make sure to pass on *what* data they selected

Please also note that tracking off-page clicks can be somewhat tricky using this, so be mindful of this.

### PR DESCRIPTION
**Pull request by etf-yrm**
Added function to track clicks on buttons and transfer parameters to GA.
For tracking you need:
- Assign tracking by identifier or other attribute, like this: clickview('identifier');
- You can pass a custom name parameter by adding an attribute to the tag data-cv_name = "custom name" , if the custom name is not used, the button text will be used as the name.
-The same parameters are transmitted: the address of the click page, if this is a link, then the address of the page where the button leads.
- Tracking has already been added for the Sign Up and Log In buttons.

fixes problem #92 
closes #92

### CODE REVIEW

File: _includes/analytics.html [hunk 1]
```diff
@@ -7,5 +7,8 @@
 
 ga('create', '{{ site.google_analytics }}', 'auto');
 ga('send', 'pageview');
+
+function clickview(idview){$(idview).bind('click',function(e){var b=$(this), type=b.attr('data-cv_active'), name=b.text(), h_from=window.location.href, h_to=false; if(b[0].nodeName=='A'){var h_to=b.attr('href');}; if(b.attr('data-cv_name')!==undefined){var name=b.attr('data-cv_name');}; ga('send', 'event', 'clickbutton',name, 'click_from:'+h_from+', click_to:'+h_to); if(h_to!==false){setTimeout(function(){if(b.attr('target')!==undefined){if(b.attr('target')!=='_blank'){window.open(h_to,'_self');}else{window.open(h_to);}}else{window.open(h_to,'_self');}},10); return false;};});};
```
Comment (vpetersson) [inline _includes/analytics.html]: `data-cv_name`?

**Comment by vpetersson** [comment]
@etf-yrm How would you be able to use this to determine which page someone was on when they clicked 'Signup'?

**Comment by etf-yrm** [comment]
@vpetersson i use window.location.href

File: _includes/analytics.html (hunk 1 above)
Comment (etf-yrm) [reply _includes/analytics.html]: for add custom name in tag for clickview function

Comment (vpetersson) [reply _includes/analytics.html]: Yes, I get that, but the variable name is very strange. Let's just call it `ga-track-name` if so as that is much more descriptive of what it is.

Comment (etf-yrm) [reply _includes/analytics.html]: @vpetersson attribute rename

**Pull request merged**